Simplenote/SPTableView.m

```diff
@@ -24,24 +29,78 @@ - (void)keyDown:(NSEvent *)theEvent
     [super keyDown:theEvent];
 }
 
+- (BOOL)isRowClipped:(NSInteger)row
+{
+    if (row < 0) {
+        return NO;
+    }
+
+    NSRect rectOfRow = [self rectOfRow:row];
+    NSPoint rowInWindow = [self convertPoint:rectOfRow.origin toView:nil];
+    CGFloat windowHeight = self.window.contentLayoutRect.size.height;
+
+    return rowInWindow.y >= windowHeight;
```

Comment:
If we were to do `return (rowInWindow.y - rectOfRow.size.height) >= windowHeight;` seems like it might let the row be partially clipped and still avoid the jump?

Reply:
Replied below, thank you sir!